### DISCUSSION TITLE
Elastic agent custom log fields not showing in dashboards

### DISCUSSION THREAD

**Discussion by alveare1** [Security-Onion-Solutions/securityonion#12670] [2.4]
### Version

2.4.60

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8 to 16 depending on node

### RAM

16 or 32gb

### Storage for /

512

### Storage for /nsm

512

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I am importing a custom log using the csv processor to separate the fields. 

I added the field event.category  and set to network but any of the prebuilt dashboards are not seeing it.
same goes for agent.name

I am not understanding why a custom log data with the correct ECS field names are not showing in the dashboard.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment]
Can you share an example parsed log?
